pkg/controllers/user/helm/common/common.go

```diff
@@ -78,7 +79,9 @@ func InstallCharts(rootDir, port string, obj *v3.App) error {
 	commands := make([]string, 0)
 	commands = append([]string{"upgrade", "--install", "--namespace", obj.Spec.TargetNamespace, obj.Name}, setValues...)
 	commands = append(commands, rootDir)
-
+	if obj.Status.ForceUpgrade {
```

Comment:
you should change the status to false after force-upgrade is done. Otherwise every install will be treated as a force-upgrade.

Reply:
`forceUpgrade` defaults to `false` for each new upgrade or rollback, so every install will be treated as non force-upgrade if a user didn't specify `forceUpgrade=true`.